### ISSUE TITLE
App url?

### ISSUE THREAD

**Issue by dmcglinn** [MoBiodiv/mobsim_app#2]
We need to add the app url (where it is being served) to the readme also this should be linked in to the mobsim repo as well

**Comment by dmcglinn** [comment]
@KatharinaGerstner @FelixMay do you know the url of where the app is being hosted? It seems the old link: http://idiv-app1.inf-bb.uni-jena.de:8080/mobsim_app/ is not working anymore.

**Comment by dmcglinn** [comment]
It looks like @AlbanSagouis may have posted it here: https://sagouis.shinyapps.io/mobsim_app/ Should we come up with a more official host of the app?

**Comment by jmchase** [comment]
Hey Dan, I think this conversation is premature.  The mobsim app is still very much in development phase and not ready for primetime.  When it is, we'll roll it out in a more official capacity. At least that's my take. Alban is still working on the guts of the thing, and it needs lots of work and I don't think that's public yet.

**Comment by dmcglinn** [comment]
Gotcha! It seems like he is really adding some very cool functionality. Do you think iDiv will want to host the older published version of the app for the time being? If not then I would like to set one up on my end for teaching purposes. Thanks!

**Comment by jmchase** [comment]
Yeah, honestly, I'm not sure what happened to the link they had online. But that's a good point, I had forgotten.  Did we actually link to it in the MEE paper?  If so, I guess we need to find out what happened to it. To be honest, though, I'd rather not work with these particular people at iDiv. There's a likelihood that some will not be renewed in their positions in the next 7 months and they've been a bit of a mess.  And so I think we need to find a new home for it anyway....  I guess it's best if you try to do it, unless you think we should try something

**Comment by AlbanSagouis** [comment]
Hi all.
@jmchase, the app is actually publicly available on my shinyapps page and I can't make it private with the free plan. If this is an issue, I can disconnect it completely.

**Comment by jmchase** [comment]
@AlbanSagouis No, I think it's fine to be public.  I just wouldn't advertise it too widely yet, until we're ready to unveil the magic you've done.
